spec/features/cwic/granules/cwic_granule_exclusion_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "spec_helper"
+
+describe "CWIC-enabled granule results", reset: false do
```

Comment:
Please add tests for:

* Undoing a removal
* Re-adding granules from to the list
* Proceeding to data access page (ensuring that granules stay removed)